tests/security/yast2_apparmor/manually_add_profile.pm

```diff
@@ -52,7 +52,7 @@ sub run {
     # it should be failed
     assert_and_click("AppArmor-Manually-Add-Profile", timeout => 60);
     send_key "alt-l";
-    assert_screen("AppArmor-Chose-a-program-to-generate-a-profile", timeout => 90);
+    send_key_until_needlematch("AppArmor-Chose-a-program-to-generate-a-profile", "alt-n", 30, 3);
```

Comment:
The performance issue is only on aarch64, right?  Is the VR already running on other platforms?

Reply:
> The performance issue is only on aarch64, right? Is the VR already running on other platforms?

Yes, I only found this issue happened on aarch64. 
I think it is enough only run on aarch64 according to the code changes.
And I have run 2 times and found no random failure again.
I also added one new needle,the new needle also fix the failure (if the needle matches, there will no the page appears https://openqa.suse.de/tests/5051800#step/manually_add_profile/29 ), and I have run with the new needle 3 times and they are passed.

Reply:
Good to know, thanks for the clear explanation. LGTM.